ZividOpenCV/CaptureUndistortRGB/CaptureUndistortRGB.cpp

```diff
@@ -12,9 +12,15 @@ Undistort an RGB image from a ZDF point cloud using Zivid camera intrinsics.
 
 #include <iostream>
 
-cv::Mat pointCloudToRGB(const Zivid::PointCloud &);
+cv::Mat pointCloudToBGR(const Zivid::PointCloud &);
+cv::Mat imageToBGR(const Zivid::Image<Zivid::RGBA8> &);
 std::tuple<cv::Mat, cv::Mat> reformatCameraIntrinsics(const Zivid::CameraIntrinsics &);
-void displayRGB(const cv::Mat &, const std::string &);
+void displayBGR(const cv::Mat &, const std::string &);
+std::string getInput(void);
+
+cv::Mat getImage2D(Zivid::Camera &camera);
+
+cv::Mat getImage3D(Zivid::Camera &camera, Zivid::Application &zivid);
```

Comment:
Is there a reason why you have a blank line between some of the forward declarations but not all?

Reply:
no, used Visual Studio 2017 to refactor ;) didn't correct it